src/lib.rs

```diff
@@ -605,7 +605,8 @@ impl<'a> UncompressedHeader for BitWriter<'a, BE> {
         self.write_bit(false)?; // no timing info present
         self.write(5, 0)?; // one operating point
         self.write(12,0)?; // idc
-        self.write(5, 0)?; // level
+        self.write(5, 31)?; // level
+        self.write(1, 0)?; // tier
```

Comment:
Seems that tier is not written to the header.

Reply:
The condition is actually whether level is >7:
https://aomediacodec.github.io/av1-spec/#sequence-header-obu-syntax

Reply:
I see. I was looking at the reduced still picture header area, my bad.

Reply:
I see. I was looking at the reduced still picture block, which does not code tier values in any case. My bad.